### PR TITLE
Kim/goth 394

### PR DESCRIPTION
**Pull request by kimlarocca**

### CODE REVIEW

File: assets/scss/extensions.scss [hunk 1]
```diff
@@ -306,6 +306,10 @@ $img-height-mobile: 106px;
     gap: 1rem !important;
   }
 }
+.v-card,
```
Comment (bonomite) [inline assets/scss/extensions.scss:309]: this breaks the "single-story-feature" on the home page.

File: pages/search.vue [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  //   event_category: 'Click Tracking',
+  //   component: 'Footer',
+  //   event_label: 'Become a member',
+  // })
+}
+</script>
+<template>
+  <div>
+    <div class="search-page">
+      <div class="pb-4">
```
Comment (bonomite) [inline pages/search.vue:45]: don't need the pb-4

File: pages/search.vue [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+        <section>
+          <div class="content">
+            <div class="grid gutter-x-xl">
+              <div class="col-1 hidden xl:block"></div>
+              <div class="col">
```
Comment (bonomite) [inline pages/search.vue:50]: image

Comment (bonomite) [reply pages/search.vue:50]: this uses the grid to lay out... and you can remove the max-width and width from ".search form"

Comment (bonomite) [reply pages/search.vue:50]: image

File: pages/search.vue [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+                <div class="search-page-results pt-2">
+                  <span v-if="articles">
+                    <strong>{{ articles.length }}</strong> results for:
+                  </span>
+                </div>
+                <form id="search" class="mt-4 mb-2">
+                  <input
```
Comment (bonomite) [inline pages/search.vue:57]: consider initially focusing on the input text field

File: pages/search.vue [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    line-height: var(--font-size-17);
+    transition: var(--transition-duration);
+    &:focus {
+      outline: none;
+      border-bottom: solid 2px var(--soybean);
+    }
+  }
+  .search-page-button {
+    position: absolute;
+    right: 1.5rem;
```
Comment (bonomite) [inline pages/search.vue:186]: right: 0.5rem to match other input field design

File: pages/search.vue [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+    #f3f3e4 0,
+    rgba(255, 255, 255, 0) 720px,
+    rgba(255, 255, 255, 0) 100%
+  );
+  form {
+    max-width: 894px;
+    width: 100%;
+    position: relative;
+  }
+  .search-page-input {
```
Comment (bonomite) [inline pages/search.vue:166]: after changing the button to right: 0.5rem, add padding-right: 50px; so the text does not overlap the button

File: pages/search.vue (hunk 4 above)
Comment (bonomite) [reply pages/search.vue:57]: at <768px width, we should probably make the input font size a little smaller... I would choose 14 for size and height

**Review by bonomite — CHANGES_REQUESTED** [review]
couple small things

**Comment by kimlarocca** [comment]
@bonomite I didn't remove the max width, kelly wanted it that way

**Review by bonomite — APPROVED** [review]
k

**Pull request merged**